### DISCUSSION TITLE
How to get a wipe tower

### DISCUSSION THREAD

**Discussion by Tinchus2009** [supermerill/SuperSlicer#1379] [General]
Hello. I have a duel head machine: T0 for main material and T1 for soluble support material. What I need is to have the wipe tower for purging some material in every tool change, for primimg reason. But everytime I activate the prime tower option I get the message saying that option is only compatible with non soluble support material.
Is there any work around this? because having the option ooze prevention enables seems to not be compatible either?

**Comment by supermerill** [comment]
1) set your printer with relative E
2) activate support, set the z-offset to "soluble", and then activate "synchronize with object layer"
3) set your extruder for interface to the second one.

![image](https://user-images.githubusercontent.com/6536403/124301785-9484b300-db60-11eb-8bd8-8a1210be8310.png)
![image](https://user-images.githubusercontent.com/6536403/124301827-a6665600-db60-11eb-8944-cd77b18fd432.png)

**Reply by supermerill** [reply]
[Shape-Box-mmu.3mf.zip](https://github.com/supermerill/SuperSlicer/files/6755771/Shape-Box-mmu.3mf.zip)
